src/formatter.ts

```diff
@@ -0,0 +1,27 @@
+export const mask = (value: string) => value.replace(/([:+,?'])/g, "?$1");
```

Comment:
Ha, just noticed I made the same mistake:

The decimal notation (`,`) is not escaped because it is not a separator. I.e. you don't write `"hello?, world"` but `"hello, world"`.

Reply:
The GKV documentation (Pflege, Technische Anlage 1, p. 10) says otherwise:
> Soll eines der hier vereinbarten Trennzeichen (Doppelpunkt, Plus-Zeichen, Komma, Fragezeichen, Apostroph) innerhalb eines Feldes als Textzeichen übermittelt werden, so muss das Aufhebungszeichen vorangestellt werden.

A comma might not be a valid seperator according to Edifact, but do we know for sure they implemented their software and file formats correctly according to Edifact? I prefer to stick with the GKV documentation. It is probably wrong, but it has been used as reference for this domain since 19 years. 

But it's good that you pointed that out. It is definitely an inconsistency that we need to keep an eye on.
We should insert a text with comma in our sample data. Let's see if they identify it as error when testing it. 😉

Reply:
Oof. That would mean that if an element is documented to be a decimal, `"3,4"` is okay, but if it is documented to be any string, it must be `"3?,4"`. Doesn't really make sense, I suspect it is a mis-documentation, so I think we should ask them.

In the Kostenträger file, there are no `,` at all, so not possible to check. Are there maybe any example-files for the invoices?

Just FYI, this is the relevant paragraph from the documentation:
```
                         Reserved for use as:
Apostrophe             ' segment terminator
Plus sign              + segment tag and data element
                         separator
Colon                  : component data element separator
Question mark          ? release character

?  immediately preceding one of the characters ' + : ?
restores their normal meaning. E.g. 10?+10=20 means
10+10=20. Question mark is represented by ??.
```
https://unece.org/fileadmin/DAM/trade/edifact/untdid/d422_s.htm

Reply:
There are no example files I am aware of yet.